static/gsApp/views/amCheckout/steps/productSelect.tsx

```diff
@@ -126,41 +142,31 @@ function ProductSelect({
                       </Fragment>
                     ) : (
                       <Fragment>
-                        <IconAdd />{' '}
-                        {formatCurrency(
-                          utils.getReservedPriceForReservedBudgetCategory({
-                            plan: activePlan,
-                            reservedBudgetCategory: productInfo.apiName,
-                          })
-                        )}
-                        /{billingInterval}
+                        <IconAdd />
+                        {tct(' [cost]/[billingInterval]', {
+                          cost,
+                          billingInterval,
+                        })}
                       </Fragment>
                     )}
                   </ButtonContent>
                 </Button>
+              </Row>
+              <Row style={{justifyContent: 'center'}}>
```

Comment:
instead of overriding it like this, we should pass an argument into Row, then the css would look like this

```
const Row = styled('div')<{justifyContent?: string}>`
  display: flex;
  gap: ${space(4)};
  justify-content: ${p => p.justifyContent ?? 'space-between'};
`;
```

Reply:
sounds good i'll make sure to remember that 😁